onward/app/feed/DeeplyReadAgent.scala

```diff
@@ -104,51 +101,40 @@ class DeeplyReadAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi) ex
           .showFields("all")
           .showReferences("all")
           .showAtoms("all")
```

Comment:
as a quick win maybe we just set `.showReferences("none").showAtoms("none")` or whatever the syntax is